Seneca-CDOT/telescope
# [Issue-3666] Add PWA to docusaurus

## Issue This PR Addresses
Fixes #3666


## Type of Change




## Description
- Add PWA to docusaurus


## Steps to test the PR


- Checkout branch
- Run `pnpm i`
- Run `pnpm --filter @senecacdot/telescope-docs build && pnpm --filter @senecacdot/telescope-docs serve`

Should able to see this button to use PWA
![image](https://user-images.githubusercontent.com/58233223/194449708-b19eece7-f581-4703-a566-4a360e85db5c.png)


## Checklist
